applications/HDF5Application/python_scripts/utils.py

```diff
@@ -0,0 +1,58 @@
+'''HDF5 process utilities.
+
+license: HDF5Application/license.txt
+'''
+import KratosMultiphysics
+
+
+def IsDistributed():
+    return KratosMultiphysics.ParallelEnvironment.GetDefaultDataCommunicator().IsDistributed()
+
+
+def InsertSingleSetting(dest, key, parameters):
+    if dest.Has(key):
+        dest[key] = parameters
+    else:
+        dest.AddValue(key, parameters)
+
+
+def InsertSettings(src, dest):
+    for key, parameters in src.items():
+        InsertSingleSetting(dest, key, parameters)
+
+
+def InsertArrayOfSettings(src, dest):
+    for current_src, current_dest in zip(src, dest):
+        InsertSettings(current_src, current_dest)
```

Comment:
for the three functions above you can look into #3889 , could you use this instead?